feature/system/gnmi/metadata/tests/large_set_consistency_test/README.md

```diff
@@ -29,6 +29,8 @@ dut.testbed
     * stop sending REQUEST requests
     * wait 5 seconds
     * SubscribeRequest once for configuration root-level metadata and verify that it is identical corresponding root-level metadata of 2nd_LARGE_CONFIGURATION.
+* Push a Large metadata of size 100KB and wait for SetResponse that confirm push was sucesfull.
```

Comment:
The specification for metadata states ([here](https://github.com/openconfig/public/blob/master/release/models/extensions/openconfig-metadata.yang#L36)) states that it can be up to 1MiB in size. It seems prudent for us to determine whether that contract is being met, can we make a table-driven test that covers some additional sizes?